src/pls/models/composables/type.py

```diff
@@ -146,7 +146,7 @@ def populate_dest(self):
             # ``dest_node`` is type ``Node``.
             link.lstat()
             self.dest_node = type(self.node)(name=link_path, path=link, is_pseudo=True)
-        except (FileNotFoundError, OSError, RuntimeError) as exc:
+        except (OSError, RuntimeError) as exc:
```

Comment:
Just curious, why did you remove `FileNotFoundError`?

Reply:
FileNotFoundError is a subclass of OSError, no need to handle them both.  https://stackoverflow.com/a/28633573